tardis/transport/montecarlo/montecarlo_main_loop.py

```diff
@@ -72,6 +75,10 @@ def montecarlo_main_loop(
     vpacket_collections = List()
     # Configuring the Tracking for R_Packets
     rpacket_trackers = List()
+
+    # Creating an empty List where individual RPacketLastInteractionTracker
+    # class will be stored.
```

Comment:
I feel this comment is redundant and only duplicates the code. See e.g. https://stackoverflow.blog/2021/12/23/best-practices-for-writing-code-comments/ rule 1

Since this isn't strictly enforced, this is only a minor comment.